src/appleseed.cli/main.cpp

```diff
@@ -327,13 +327,62 @@ namespace
             params.insert("noise_seed", noise_seed);
         }
 
+        if (g_cl.m_output.is_set())
+        {
+            const char* file_path = g_cl.m_output.value().c_str();
+            params.insert("output_path", file_path);
+        }
+
+        if (g_cl.m_checkpoint_create.is_set())
+        {
+            params.insert("checkpoint_create", "on");
+
+            if (g_cl.m_checkpoint_create.values().empty() && !g_cl.m_output.is_set())
+            {
+                LOG_ERROR(g_logger, "ouput path must be specified if no path is given for --checkpoint-create");
```

Comment:
Typo: *output*

Reply:
Can you retrieve the actual option name instead of repeating it in the error message?

Reply:
I could.